### PR TITLE
Adding in the new Sci-borg

### PR DESCRIPTION
**Pull request by kcin2001**
This pr is almost complete but it is still unfinished

Required
- [x] the dogborg sprites to be completed from valekin
- [ ] add last dogborg
Optional
- [ ] figure out how to make the belly and destructive analyzer play nice as one item

The sprites
image
image
image

### CODE REVIEW

**Comment by Screemonster** [comment]
Marking DNM on request 'cause it's WIP.

**Comment by kcin2001** [comment]
thank you scree

**Comment by kcin2001** [comment]
the do not merge label can be removed now

**Comment by Arokha** [comment]
Conflict, sorry, also what did you end up doing with the DA? Is it just the normal robot DA for now?

File: code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm [hunk 1]
```diff
@@ -140,6 +140,16 @@
 	combat = 1
 	attack_verb = list("batted", "pawed", "bopped", "whapped")
 	chargecost = 500
+	
+/obj/item/weapon/shockpaddles/robot/hound/jumper
+	name = "paws of life"
+	icon = 'icons/mob/dogborg_vr.dmi'
+	icon_state = "defibpaddles0"
+	desc = "Zappy paws. For fixing cardiac arrest."
```
Comment (Arokha) [inline code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm]: If these are the jumper cable versions, definitely change the name and description.

File: code/modules/mob/living/silicon/robot/robot_modules_vr.dm [hunk 2]
```diff
@@ -147,3 +149,57 @@
...
+	src.modules += new /obj/item/weapon/gripper/no_use/mech(src)
+	src.modules += new /obj/item/weapon/gripper/no_use/loader(src)
+	src.modules += new /obj/item/device/robotanalyzer(src)
+	src.modules += new /obj/item/weapon/card/robot(src)
+	src.modules += new /obj/item/weapon/weldingtool/electric/mounted/cyborg(src)
+	src.modules += new /obj/item/weapon/screwdriver/cyborg(src)
+	src.modules += new /obj/item/weapon/wrench/cyborg(src)
+	src.modules += new /obj/item/weapon/wirecutters/cyborg(src)
+	src.modules += new /obj/item/device/multitool(src)
+	src.modules += new /obj/item/weapon/reagent_containers/syringe(src)
```
Comment (Arokha) [inline code/modules/mob/living/silicon/robot/robot_modules_vr.dm]: Why syringe?

File: code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm [hunk 3]
```diff
@@ -496,6 +496,15 @@
...
 	var/max_item_count = 25
 
+/obj/item/device/dogborg/sleeper/analyzer //sci-borg gut.
+	name = "Dogborg Digestive Analyzer"
+	desc = "A mounted destructive analyzer unit with fuel processor."
+	icon = 'icons/mob/dogborg_vr.dmi'
+	icon_state = "analyzer"
+	inject_amount = 10
+	min_health = -100
+	injection_chems = list("inaprovaline") //So they don't have all the same chems as the medihound!
```
Comment (Arokha) [inline code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm]: Why give them inaprov? They should probably have none like the janipup.

**Comment by kcin2001** [comment]
yes they have the normal destructive analyzer for now, they have syringe because normal rnd borgs have a syringe which i assume is for xenobio, and innap was suggested by vale to help keep unlucky scienctists not dead as they are carted to med but i can remove it if you want

**Comment by Arokha** [comment]
Would rather remove the innap yeah. It'll 1) keep them from duplicating too much of the other borg features (which I consider ESPECIALLY important on this one since it has like literally fucking everything) and 2) will clean up the UI slightly by removing the top injector parts.

**Comment by kcin2001** [comment]
what items do you suggest the borg to keep and to remove

**Comment by Arokha** [comment]
We discussed and think it should just be a scientist dogborg basically. The goal is to have the borg be basically one of the titles in the department, one per department. They're already a little more powerful than normal borgs because of their generic doggo tools (bite, nose, tongue, sleeper), so making sure they don't overlap a bunch of titles is important to us. K9 is an officer, Medihound is basically an EMT, Janipup is trashdog.

So basically only these items:
- Normal dog items
- Portable DA
- Research gripper
- Beaker (acid for circuit printers)
- RPED (and a screwdriver required to use it in most machines)
- Emag-only hand tele (why not, I guess)

**Comment by kcin2001** [comment]
should i make it unable to help in xenobio?

**Comment by Arokha** [comment]
For now, yeah. We are debating having a separate model. Just the items above please for this PR though.

**Comment by kcin2001** [comment]
ill leave most of the stuff there but commented out for easier refrence

**Comment by kcin2001** [comment]
you could make at least 3 models out of the rnd borg
standard
robotics
xeno

**Comment by kcin2001** [comment]
and arokha would you be able to fix the conflict because i dont know how to

**Comment by kcin2001** [comment]
@Arokha did i fulfill your requested changes?

File: code/modules/mob/living/silicon/robot/robot_modules_vr.dm [hunk 4]
```diff
@@ -147,3 +149,53 @@
...
+	var/datum/matter_synth/nanite = new /datum/matter_synth/nanite(10000)
+	synths += nanite
+	var/datum/matter_synth/wire = new /datum/matter_synth/wire()						//Added to allow repairs, would rather add cable now than be asked to add it later,
+	synths += wire																		//Cable code, taken from engiborg,
+
+	var/obj/item/stack/nanopaste/N = new /obj/item/stack/nanopaste(src)
+	N.uses_charge = 1
+	N.charge_costs = list(1000)
+	N.synths = list(nanite)
+	src.modules += N
```
Comment (Arokha) [inline code/modules/mob/living/silicon/robot/robot_modules_vr.dm]: The infinite wire, nanopaste, and cable coils should be removed as well. Wouldn't be needed by a normal scientist borg.

Comment (SpadesNeil) [reply code/modules/mob/living/silicon/robot/robot_modules_vr.dm]: What of robotics repair? Although I guess this may cause for a self repairing borg.

Comment (kcin2001) [reply code/modules/mob/living/silicon/robot/robot_modules_vr.dm]: Well even if it did still have the card i dont know if it could open itself

Comment (Arokha) [reply code/modules/mob/living/silicon/robot/robot_modules_vr.dm]: @SpadesNeil Per our discussion in the admin chat, we decided we didn't want it doing robotics, hence asking for the removal.

**Comment by kcin2001** [comment]
though this is now not ready to merge again as i need to add in another borg that has been made

**Comment by Arokha** [comment]
Might be ready, but can't merge until you fix that conflict. S'not something I can do for you since don't have write access to your stuff.

**Comment by kcin2001** [comment]
I dont know how to

**Comment by Arokha** [comment]
All the custom vorestation cyborg modules moved from `robot_modules_vr.dm` to `robot_modules/station_vr.dm`

You'll have to copypaste your additions to that file into the new one. I don't know what git client you're using, but you could try to pull the current vorestation master branch into your working branch and it will prompt you to fix the conflicts. At that point you'd go in and copy the things out of robot_modules.dm into station_vr.dm, and then delete robot_modules.dm and finish the merge.

**Comment by kcin2001** [comment]
i use the web client since the app version is beyond me and i can only use it on one of my 2 computers

**Comment by kcin2001** [comment]
ugh that didnt work right

**Comment by kcin2001** [comment]
i dont know how to fix it
